andy_li/Python Fundamentals/Names.py

```diff
@@ -0,0 +1,48 @@
+# # Part I
+# def names(lst):
+#     for elem in lst:
+#         line_str = ""
+#         for key, val in elem.items():
+#             line_str += val + " "
+#
+#         print line_str
+#
+#
+# students = [
+#      {'first_name':  'Michael', 'last_name' : 'Jordan'},
+#      {'first_name' : 'John', 'last_name' : 'Rosales'},
+#      {'first_name' : 'Mark', 'last_name' : 'Guillen'},
+#      {'first_name' : 'KB', 'last_name' : 'Tonel'}]
+#
+# names(students)
+
+# Part II
+def names2(dict):

```

Comment:
3 for loops in one function is terribly inefficient. This does work though. In your spare time, could you think of a better way to accomplish this without the third for loop. What about without the second for loop?